tests/simulate/test_generic.py

```diff
@@ -52,8 +53,9 @@ def test_invalid(self):
             sim = relentless.simulate.Simulation(self.ops)
             sim.run(self.ensemble, self.potentials, self.directory)
 
-    @unittest.skipIf(not relentless.simulate.lammps._lammps_found,
-                     "LAMMPS not installed")
+    @unittest.skipIf(not relentless.simulate.lammps._lammps_found or
```

Comment:
Which part of this test fails under python 3.8?

I am not sure that we should keep this test around anyway. LAMMPS has a Brownian integrator in more recent versions, so we might need to drop this.

Reply:
Ok, I have removed this test.